to-do-ui/src/app/components/task-form/task-form.component.html

```diff
@@ -0,0 +1,16 @@
+<p-toast position="top-center"></p-toast>
+<form class="flex flex-column gap-4" [formGroup]="taskForm" (ngSubmit)="onSubmit()">
+
+	<input type="text" pInputText label="Title" formControlName="title">
+
+	<textarea pInputTextarea type="text" pInputTextArea label="Description" formControlName="description" [rows]="'5'"
+		[cols]="'90'"></textarea>
+
+	<p-toggleButton [onIcon]="'pi pi-check-circle'" [onLabel]="'Completed'" [offLabel]="'Mark as done'"
+		[offIcon]="'pi pi-spinner'" [iconPos]="'right'" [style]="{'width': '10em'}" formControlName="isCompleted">
+	</p-toggleButton>
+
+	<button pButton type="submit" [label]="config.data.actionType" (click)="showToast()" class="p-button-success"
+		[disabled]="this.taskForm.value['title'] === ''"></button>
```

Comment:
```suggestion
		[disabled]="this.taskForm.value['title'] === ''"></button>
```

you can add a getter for this.taskForm.value['title']